src/lib/components/profile/carousel/Carousel.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import rightArrow from '$lib/icons/right-arrow.svg'
+  import leftArrow from '$lib/icons/left-arrow.svg'
+
+  export let images: string[]
+
+  let currentImage = 0
+
+  let nextImage = 1
+
+  let prevImage = images.length - 1
+
+  function goNext() {
+    currentImage = (currentImage + 1) % images.length
+  }
+
+  function goPrev() {
+    currentImage = (currentImage - 1 + images.length) % images.length
+  }
+
+  $: nextImage = (currentImage + 1) % images.length
+  $: prevImage = (currentImage - 1 + images.length) % images.length
```

Comment:
```suggestion
  let currentImage = 0

  $: nextImage = (currentImage + 1) % images.length
  $: prevImage = (currentImage - 1 + images.length) % images.length

  function goNext() {
    currentImage = (currentImage + 1) % images.length
  }

  function goPrev() {
    currentImage = (currentImage - 1 + images.length) % images.length
  }
```
Debiera funcionar según https://learn.svelte.dev/tutorial/reactive-declarations